specfile/changelog.py

```diff
@@ -223,6 +246,54 @@ def delete(index):
         else:
             delete(i)
 
+    def filter(
+        self, since: Optional[str] = None, until: Optional[str] = None
+    ) -> "Changelog":
+        """
+        Filters changelog entries with EVR between since and until.
+
+        Args:
+            since: Optional lower bound. If specified, entries with EVR higher
+              than this will be included.
+            until: Optional upper bound. If specified, entries with EVR lower
+              than this will be included.
```

Comment:
+1

I am just not sure if it will be handier to have `higher` or `higher or equal`. What will be better for the Packit's usecase?

Reply:
`higher` is more convenient, because we need a changelog diff since the last stable EVR. But that can be easily done with `higher or equal` and slicing. `higher or equal` is perhaps more intuitive?

Reply:
Ok, let me change that.

Reply:
Sorry for such nitpicking..;)